config/playground/jobs/sample_data_tooling/refresh_deployment.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: refresh-data-cronjob
+spec:
+  # Runs everyday at 1 am UTC
+  schedule: "0 1 * * *"
+  jobTemplate:
+    spec:
+      template:
+        spec:
+          containers:
+            - name: refresh-data-job
+              image: huyaboo/demo
+              env:
+              - name: SAMPLE_DATA_USERNAME
+                valueFrom:
+                  secretKeyRef:
+                    name: demo-secret
```

Comment:
use the good name instead of demo-secret. Thanks

Reply:
`demo-secret` has been changed to `sample-data-secret`